docs/setup/cookbook.md

```
@@ -4,3 +4,2173 @@ hidden: true
+                          admin to manage         
+                          configuration, users    
+                          and master data.        
+  -----------------------------------------------------------------------
+
+**6. Software and Platform Requirements**
+
+MOSIP supplies the platform and integrates the third-party components.
+The country team is responsible for the operating environment, gateways
+and identity-related accounts.
+
+![](.gitbook/assets/media/image2.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)